azure-dns-new-zone/azuredeploy.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newZoneName": {
+      "type": "String",
+	  "defaultValue": "example.com",
```

Comment:
We can support multiple copies of the same DNS zone but it's limited, so I can remove the default if desired.